deploy/olm-catalog/3scale-operator/3scale-operator.v0.3.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,244 @@
+      description: Binding
+      displayName: Binding
+    - kind: Limit
+      name: limits.capabilities.3scale.net
+      version: v1alpha1
+      description: Limit
+      displayName: Limit
+    - kind: MappingRule
+      name: mappingrules.capabilities.3scale.net
+      version: v1alpha1
+      description: MappingRule
+      displayName: MappingRule
+    - kind: Metric
+      name: metrics.capabilities.3scale.net
+      version: v1alpha1
+      description: Metric
+      displayName: Metric
+    - kind: Plan
+      name: plans.capabilities.3scale.net
+      version: v1alpha1
+      description: Plan
+      displayName: Plan
+    - kind: Tenant
+      name: tenants.capabilities.3scale.net
+      version: v1alpha1
+      description: Tenant
+      displayName: Tenant
+  description: |
+    The 3scale Operator creates and maintains the Red Hat 3scale API Management on [OpenShift](https://www.openshift.com/) in various deployment configurations.
+
+    [3scale API Management](https://www.redhat.com/en/technologies/jboss-middleware/3scale) makes it easy to manage your APIs.
+    Share, secure, distribute, control, and monetize your APIs on an infrastructure platform built for performance, customer control, and future growth.
+
+    ### Supported Features
+    * **Installer** A way to install a 3scale API Management solution, providing configurability options at the time of installation
+    * **Capabilities** Ability to define 3scale API definitions and set them into a 3scale API Management solution
+
+    ### Upgrading your installation
+    Currently upgrading feature is not available.
+
+    ### Documentation
+    Documentation to the current _master_ branch as well as all releases can be found on our [website](https://github.com/3scale/3scale-operator).
+
+    ### Getting help
+    TBD
+
+    ### Contributing
+    TBD
+
+    ### License
+    3scale Operator is licensed under the [Apache 2.0 license](https://github.com/3scale/3scale-operator/blob/master/LICENSE)
+
+  keywords:
+    - 3scale
+    - API
+  displayName: 3scale Operator
+  install:
+    spec:
+      deployments:
+      - name: 3scale-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: threescale-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: threescale-operator
+            spec:
+              containers:
+              - command:
+                - 3scale-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: threescale-operator
+                image: quay.io/3scale/3scale-operator:v0.3.0
```

Comment:
shouldn't this be a productized image in case of the productized branch?

Reply:
this is not productized branch